app/views/foreman/unattended/snippets/_chef_client_omnibus_bootstrap.erb

```diff
@@ -0,0 +1,10 @@
+# install.sh will install latest chef client version
+curl -L https://www.chef.io/chef/install.sh | bash
+
+# if you want specific version you can use something like this instead of install.sh
+# yum install https://opscode-omnibus-packages.s3.amazonaws.com/el/6/x86_64/chef-11.10.0-1.el6.x86_64.rpm
```

Comment:
Are there chef pckages available in common OS repositories? I see that Debian wheezy has chef 10 and jessie has 11, Ubuntu trusty has also 11 and FreeBSD 11, too.